### ISSUE TITLE
fabric.Group mousedown event on subtarget fires more than once per click

### ISSUE THREAD

**Issue by Sirozha1337** [kangax/fabric.js#3923]
## Version
1.7.11

## Test Case
https://jsfiddle.net/dmjec08b/3/

## Steps to reproduce
* Create two rectangles with diffrent color. 
* Place "mousedown" events on those objects with console.log('click on rect1') and console.log('click on rect2'). 
* Use fabric.Group to make a group and set subTargetCheck=true. 
* Place a "mousedown" event on a group with console.log('click on group'). 
* Click on rect1 multiple times.
* Compare how many times 'click on rect1' was logged and how many times 'click on group' was logged.
* 'click on group' was logged once per mouse click, 'click on rect1' was logged multiple times per click. 

## Expected Behavior
Mousedown event listener on object in fabric.Group gets callled only once per click.

## Actual Behavior
Mousedown event listener on object  in fabric.Group gets callled multiple times, when you actually made only one mouse click.

**Comment by Sirozha1337** [comment]
Found another issue with the same problem, so I'm closing this. Sorry for the inconvenience.
